### ISSUE TITLE
'Axes|Plane' only visible when 'Plane OXY' active

### ISSUE THREAD

**Issue by humdingerb** [threedeyes/STLover#20]
Shouldn't the "Plane axes" be visible as soon as 'Axes' is active?

To reproduce:
* Deactivate 'Plane OXY'
* Activate 'Axes'
* Tick 'Axes|Plane'

-> The coloured plane lines aren't visible (until you also activate 'Plane OXY')

The 'Compass' works as expected.

**Comment by threedeyes** [comment]
At the moment the axes on the plane are showing up as I intended. No plane - no axes. Do you think the logic should be changed? Should the axes (possibly all three) be displayed at the origin regardless of the OXY plane display?

**Comment by humdingerb** [comment]
> Should the axes (possibly all three) be displayed at the origin regardless of the OXY plane display?

At least that's how interpreted it, like the two menus "Axes" and "Plane OXY" are independent.

**Comment by threedeyes** [comment]
Hmmm... Maybe I should just disable (grayed) the Axis->Plane item if the OXY plane display is disabled?

**Comment by humdingerb** [comment]
Possible. But it's not clear at a glance why it's disabled and what needs to be done to get it enabled.
If possible, you may want to consider just drawing the "plane axes" even without the "Plane OXY" being activated.

**Comment by threedeyes** [comment]
The name of the item said it would be drawn on a plane. No plane, no axes. But if we make them independent, then we should change the name of the item Axes->Plane to for example Axes->Scene or Axes->World. I don't know what would be better.

**Comment by humdingerb** [comment]
Or "Axes->Origin" if they sit in the scenes origin.

I just reported this, because I had a look where "Compass" appears in the app for the translation. I dis/enabled Compass, saw what it did and then did the same with "Plane", but didn't see a difference. It wasn't obvious to me why it had no effect. Only after playing around some more I found the "Plane OXY" connection.
Just saying, de-coupling Plane OXY and Axes would save confusion for newbies. I'm not much of a 3D user though...
